src/chocolatey.console/Program.cs

```diff
@@ -145,7 +145,23 @@ private static void Main(string[] args)
                 var traceAppenderName = "{0}LoggingColoredConsoleAppender".FormatWith(ChocolateyLoggers.Trace.ToStringSafe());
                 Log4NetAppenderConfiguration.EnableDebugLoggingIf(config.Debug, verboseAppenderName, traceAppenderName);
                 Log4NetAppenderConfiguration.EnableVerboseLoggingIf(config.Verbose, config.Debug, verboseAppenderName);
-                Log4NetAppenderConfiguration.EnableTraceLoggingIf(config.Trace, traceAppenderName);
+
+                if (config.Information.IsProcessElevated)
+                {
+                    Log4NetAppenderConfiguration.EnableTraceLoggingIf(config.Trace, traceAppenderName);
+                }
+                else
+                {
+                    var logger = ChocolateyLoggers.Normal;
+
+                    if (!config.RegularOutput)
+                    {
+                        logger = ChocolateyLoggers.LogFileOnly;
+                    }
+
+                    "chocolatey".Log().Warn(logger, "Usage of the --trace option is only allowed when running from an elevated session.");

```

Comment:
Would it be an idea to add this string to the `StringResources.cs` file and reference that instead of having a bare string that we duplicate in `GetChocolatey.cs`?

Reply:
I knew that you were going to suggest that!

Am I right in saying that this doesn't exist in the Support Branch?

Reply:
Actually, I can't do this, since the protection level of the StringResources class means I can't use it in the chocoaltey.console project.

Reply:
That's a better reason not to than what we discussed in Slack 😏 .